bootstrap/README.md

```diff
@@ -50,9 +56,9 @@ If you have plugged in your speaker via the headphone jack, you may experience i
 
 8. Run a recipe
 
-    cd ~/Desktop/tjbot/recipes/intro
```

Comment:
Im not sure how good an idea this is.
The `intro recipe script` (and any other recipe) need credentials to work. If boostrap is run to get things installed as a first step, it is unlikely there will be credentials to setup to run. I feel running the set of tests serve as a good way to test the pi hardware setup. Intro might be testing things beyond this i.e. connection to Watson services.